cmd/scollector/main.go

```diff
@@ -39,6 +39,7 @@ var (
 	flagBatchSize       = flag.Int("b", 0, "OpenTSDB batch size. Used for debugging bad data.")
 	flagSNMP            = flag.String("s", "", "SNMP host to poll of the format: \"community@host[,community@host...]\".")
 	flagICMP            = flag.String("i", "", "ICMP host to ping of the format: \"host[,host...]\".")
+	flagHAProxy         = flag.String("haproxy", "", `haproxy credentials, of the format: "user:password"`)
```

Comment:
Yeah, this was part of the original design before I added the extra config lines. I'm fine with killing this as a flag. Will do so.